plugins/memory/memory_osx

```diff
@@ -35,33 +34,68 @@ if [ "$1" = "autoconf" ]; then
     fi
 fi
 
-TOTALMEM=$(sysctl hw.memsize | cut -d" " -f2)
-graphlimit=$TOTALMEM
-
 dehumanise() {
     echo "$1" | sed -e "s/K/*1024/g;s/M/*1024*1024/;s/G/*1024*1024*1024/;s/T/*1024*1024*1024*1024/"
 }
 
 if [ "$1" = "config" ]; then
         echo 'graph_title Memory'
-        echo "graph_order used wired unused"
-        echo "graph_args --base 1024 -r --lower-limit 0 --upper-limit $graphlimit"
+        echo "graph_order wired app compressed cached free swap_used swap_free"
+        echo "graph_args --base 1024 -r --lower-limit 0"
         echo 'graph_vlabel Bytes'
         echo 'graph_category system'
-        echo 'used.label used (not incl. wired)'
-        echo 'used.draw AREA'
-        echo 'used.min 0'
-        echo "used.info Used memory, not including wired"
         echo 'wired.label wired'
-        echo 'wired.draw STACK'
+        echo 'wired.draw AREA'
```

Comment:
You can use `AREASTACK` everywhere (i.e. for the first and for all subsequent entries).
This would make it easier to read such a patch.